fcc-test-suite-template-master/css/style.css

```diff
@@ -0,0 +1,24 @@
+ .nav {
+     width: 100vw;
+ }
+
+ .nav .navbar {
+     padding-left: 25px;
+     padding-right: 25px;
+     width: 100%;
+ }
+
+ @media (max-width: 600px) {
+    .nav .navbar {
+        padding-left: 15px;
+        padding-right: 15px;
+        width: 80%;
+    }
+  }
+
+ 
```

Comment:
Unexpected whitespace at end of line

Reply:
Most of the issues you've raised are valid :+1: 

- However, a common pattern in all the changes you've required as a reviewer is that you've only stated **what** the problem is, but you've not communicated to the student **why** they are seen as problems in the first place :blush: 
It is important to state **why** because it's always good to give a reason why you think a change should be made. It also educates the owner of such a project to write a better code and avoid the occurrence of such overlook :seedling: 

- Also, you need to phrase your comments politely, as suggestions instead of orders. For example:
This is a polite suggestion: _"I think it would be better to use semantic tags"_
This reads like an order: _"Use semantic tags"_

**Kindly apply the instructions above to all your inline comments and use them as a guide to reconstruct another set of comments under your previous comments.**